pex/util.py

```diff
@@ -167,7 +170,7 @@ def cache_distribution(cls, zf, source, target_dir):
                             zf.extract(name, target_dir_tmp)
 
         dist = DistributionHelper.distribution_from_path(target_dir)
-        assert dist is not None, "Failed to cache distribution ".format(source)
+        assert dist is not None, "Failed to cache distribution: {} ".format(source)
```

Comment:
Bug

Reply:
i came here to see if this was still an issue, glad to see it's fixed!